index.js

```diff
@@ -1,21 +1,35 @@
+const findUp = require('find-up')
 const fs = require('fs')
-const readPkg = require('read-pkg-up')
 const truncate = require('cli-truncate')
 const wrap = require('wrap-ansi')
 const pad = require('pad')
-const path = require('path')
 const fuse = require('fuse.js')
-const homeDir = require('home-dir')
 const util = require('util')
 
+const readFile = util.promisify(fs.readFile)
+
 const types = require('./lib/types')
 
-const defaultConfig = {
-  types,
-  symbol: false,
-  skipQuestions: [''],
-  subjectMaxLength: 75,
-  conventional: false
+function loadConfigFrom(filename) {
+  return findUp(filename)
+    .then(file => readFile(file, 'utf8'))
+    .then(JSON.parse)
+    .then(obj => obj && obj.config && obj.config['cz-emoji'])
+    .catch(() => null)
+}
+
+async function getConfig() {
+  const defaultConfig = {
+    types,
+    symbol: false,
+    skipQuestions: [''],
+    subjectMaxLength: 75,
+    conventional: false
+  }
+
+  const config = (await loadConfigFrom('package.json')) || (await loadConfigFrom('.czrc'))
```

Comment:
I feel like it's still necessary to look explicitly in the home directory like `readFromGlobalCzrc` was doing, no?

Reply:
From my experience, most projects sit under the home directory, at least on *NIX systems. So the current logic should capture the home directory in most cases. However, I don't really know what's the tendency on Windows systems 🤔 

In any case, it is pretty simple to implement so I'll just add it. Thanks for the raising it!

Reply:
From my experience it's something very personal that cannot be assumed. Thanks for fixing it so quickly! Really appreciate it!